pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -320,6 +320,7 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 
 	// Initialize CPU manager
 	cm.cpuManager, err = cpumanager.NewManager(
+		klog.Background(),
```

Comment:
`NewContainerManager` is called from the code that has a context. Would it make sense to pass it to the `NewContainerManager ` and use here instead of klog.Background() ?

Reply:
Yes, this is top level wiring we can do, but I will do in a different commit in this PR